app/models/avalon_jobs.rb

```diff
@@ -49,10 +49,16 @@ def self.delete_masterfile pid
       logger.warn "MasterFile #{oldpath} does not exist"
     end
   end
+
+  def self.collection_batch_ingest(collection_id)
+    collection = Admin::Collection.find(collection_id)
+    Avalon::Batch::Ingest.new(collection).ingest
+  end
+
 class << self
```

Comment:
I'm going to leave this, although I hate it too. The delayed_job docs (and my own experimenting) confirm that the only syntax that works is "class << self", which I usually shy away from. Converting means that I have to re-indent each line of code in the file, which isn't good for "git blame" and could potentially introduce errors (not likely, but possible). In Avalon 6, they move to the ActiveJob interface (as you noticed), so I would recommend dealing with this at that time.